src/frontend/apps/bbb/DashboardMeeting/index.tsx

```diff
@@ -44,9 +46,81 @@ const messages = defineMessages({
   },
 });
 
+const bbbTheme: ThemeType = deepMerge(theme, {
+  global: {
+    font: {
+      family: 'Roboto-Regular',
+    },
+  },
+  box: {
+    extend: null,
+  },
+  button: {
+    default: {
+      background: { color: 'white' },
+      border: { color: 'brand', size: 'xsmall' },
+      color: 'brand',
+      padding: { vertical: 'xsmall', horizontal: 'small' },
+    },
+    primary: {
+      background: { color: 'brand' },
+      border: undefined,
+      color: 'white',
+    },
+    border: {
+      radius: '4px',
+    },
+    size: {
+      large: {
+        border: {
+          radius: '6px',
+        },
+        pad: {
+          horizontal: '3rem',
+          vertical: '1rem',
+        },
+      },
+    },
+    extend: null,
+  },
+  formField: {
+    label: {
+      size: '0.8rem',
+      margin: '0.5rem 1rem 0',
+      color: 'bg-grey',
+    },
+    border: {
+      position: 'outer',
+      side: 'all',
+      color: 'blue-active',
+      style: 'solid',
+    },
+    round: {
+      size: 'xsmall',
+    },
+  },
+  textInput: {
+    extend: 'padding: 0 1rem 0.8rem',
+  },
+  maskedInput: {
+    extend: 'padding: 0 1rem 0.8rem',
+  },
+  dateInput: {
+    // extend: 'padding: 0 1rem 0.8rem',
+    icon: {
+      size: '18px',
+    },
+  },
+});
+
 const DashboardMeeting = () => {
   const intl = useIntl();
-  const canUpdate = getDecodedJwt().permissions.can_update;
+  let canUpdate: boolean;
+  try {
+    canUpdate = getDecodedJwt().permissions.can_update;
+  } catch (e) {
+    return <DashboardMeetingError />;
+  }
```

Comment:
In which case you can have an exception here ?

Reply:
If I remember correctly, it is in case of a get request.